float32 pan
float32 tilt
int16 point_count
